### DISCUSSION TITLE
Exporting property arrays as a list style text file from MF6 models for Pyemu

### DISCUSSION THREAD

**Discussion by laat0003** [pypest/pyemu#479] [Q&A] [answered]
Hi,

Prior to using Pyemu I set the MF6 packages that I want to have external input via Flopy's package.set_all_data_external() option. This produces list style output for stress period data in boundary condition packages but defaults to array formatted text files for property arrays. Unless I am mistaken, Pyemu appears to require property arrays in a list format text file, correct?  I'm currently getting around this via another script that rewrites the external files into a list format but I figured there must be a way to do this via Flopy. Any help on this issue would be appreciated. Apologies for posting this, which is more of a Flopy question, but I figured other Pyemu users may have the same issue.

Cheers,
Tariq

**Comment by rhugman** [comment, marked as answer]
Hey Tariq -`pyemu` handles array-style files, as long as they have consistent row x cols. Flopy-written files are not an issue if you are using DIS grids. But DISV/DISU "array" files are. 

I think the general approach rn is the one you are taking: blast through `flopy` 's generated files and re-write them in a clean list. (maybe someone can provide a better approach...).

**Reply by laat0003** [reply]
Thanks Rui,

I am working with DISV array files. I'm glad re-writing a clean list is a common approach because I thought I must be missing something obvious.  Wasted a few hours looking through flopy, GMDSI, pyemu and pypestutils  example notebooks to see if there was a different way but...nada.
